python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -973,6 +973,17 @@ def aten〇index〇Tensor_hacked_twin〡shape(self: List[int], indices: List[Lis
 def aten〇cat〡shape(tensors: List[List[int]], dim: int = 0) -> List[int]:
     return upstream_shape_functions.cat(tensors, dim)
 
+def aten〇stack〡shape(tensors: List[List[int]], dim: int = 0) -> List[int]:
+    unsqueezed_tensors: List[List[int]] = []
+    for tensor in tensors:
+        tensor_rank = len(tensor) + 1
+        assert -tensor_rank <= dim < tensor_rank, "Dimension out of range"
+        insert_dim = dim if dim >= 0 else dim + tensor_rank
+        unsqueezed = upstream_shape_functions._copy(tensor)
+        unsqueezed.insert(insert_dim, 1)
+        unsqueezed_tensors.append(unsqueezed)
+    return upstream_shape_functions.cat(unsqueezed_tensors, dim)
```

Comment:
Can you upstream this function first? See: https://github.com/pytorch/pytorch/pull/76889

Reply:
Okay, will need some time to do this.

Reply:
That took longer than expected on the PyTorch side, but they finally merged it!